### DISCUSSION TITLE
Exclude project from Monitoring?

### DISCUSSION THREAD

**Discussion by steveruizok** [vercel/community#1981] [Idea]
### Summary

Hi, question about monitoring. 

We have a project that we were using for checking whether to turn on maintenance mode. It received a lot of hits to a server-less endpoint, however we really don't care about any monitoring data related to this project. Can we exclude this project from our team's monitoring usage?

**Comment by amyegan** [comment]
Thanks for reaching out, Steve. It is not currently possible to disable Monitoring for a specific project, but I shared your feedback with the team to consider for future development
